globals/constants.go

```diff
@@ -11,8 +11,48 @@ const (
 	PROJECTKEY                       = "NTNDNDkwODItODdEMS0yRjlGLTg2MEYtNzZENjgwRDRBNzQz"
 	PROJECTSECRET                    = "ODMyQTlCNjUtNUUwMC1DOUM1LUY0MDItMTAzQThEOEM0RUQ1"
 	ZENDESKURL                       = "https://remot3it.zendesk.com/api/v2"
+	GRAPHQL_API_URL                  = "https://api.remote.it/v1/graphql"
 	RECONNECT_TIMEOUT_IN_MILISECONDS = 2 * 60 * 1000 * time.Millisecond // default is 2 min
 	SCHANNEL_BUFFER_SIZE             = 1048576                          // default is 1M
 	SCHANNEL_MESSAGE_SIZE            = 65507                            // default is 64k
 	CONTROL_WS_ENDPOINT              = 64000
+
+	PROXY_CREATE_WAIT       = "true"
+	PROXY_CREATE_ISOLATE    = "domain=app.remote.it" // FIXME:  for now this is like this, but in portal it is taken from browser URL
+	PROXY_CREATE_CONCURRENT = true                   // INFO: pass this JSON flag to API to enable concurrent proxies, FIXME: will this be remvoed in the future ?
+
+	// These are here for two reasons
+	// 1 - error code is a string with no standard
+	// 2 - is part of the `resp.Reason` instead of `resp.Error`,
+	//     `resp.Reason` must be a human readable form of the `resp.Error`
+	API_ErrorCodes_Reason_DeviceNotFound        = "[0806]"
+	API_ErrorCodes_Reason_DuplicateName         = "[0807]"
+	API_ErrorCodes_Reason_ServiceNotFoundForUID = "[0861]"
+	API_ErrorCodes_Reason_BadDeviceAddress      = "bad device address"
+	API_ErrorCodes_Reason_MissingApiToken       = "missing api token"
+	API_ErrorCodes_Reason_UserOrPassInvalid     = "username or password are invalid"
+	API_ErrorCodes_Reason_MissingUser           = "missing user"
+	API_ErrorCodes_Reason_NoMatchingBulkProject = "no matching bulk project"
+	API_ErrorCodes_Reason_MFA_1                 = "SMS_MFA"
+	API_ErrorCodes_Reason_MFA_2                 = "SOFTWARE_TOKEN_MFA"
+	API_ErrorCodes_Reason_MFA_3                 = "MFA_SETUP"
+	API_ErrorCodes_Status_False                 = "false"
+	API_ErrorCodes_Status_True                  = "true"
+	API_ErrorCodes_Status_Pending               = "pending"
+	API_ErrorCodes_Status_Reset                 = "reset"
+
+	// ConnectD | Muxer | Demuxer
```

Comment:
```suggestion
	// connectd | Muxer | Demuxer
```

The name of the binary is `connectd` not `ConnectD`